docs/Troubleshooting/Troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+description: Frequently asked questions FAQ and answers for troubleshooting Pantheon
+<!--- END of page meta data -->
+
+# Troubleshooting
+
+If Pantheon is not working as expected, here are some things to check or try. 
+
+## Command Line Options
+
+Characters such as smart quotes and long (em) hyphens won't work in Pantheon command line options. Ensure quotes have
+not been automatically converted to smart quotes or double hyphens combined into em hyphens.
+
+## Missing Command Line Options 
+
+When running Pantheon from the Docker image a number of [command line options are not used](../Getting-Started/Run-Docker-Image.md#command-line-options). 
+
+## Genesis Milestone 
+
+Not including a sufficient milestone in the genesis file can lead to unexpected and inconsistent behaviour without
+specific errors. Ensure a sufficient milestone is included in the genesis file (for example, `constantinoplefixblock`). 
+
+## Logging 
+
+Restart Pantheon with the command line option [`--logging=DEBUG`](../Reference/Pantheon-CLI-Syntax.md#logging) and look at the log files. 
```

Comment:
Do you want to go all out and use TRACE (not just debug)?

Reply:
Fixed